pkg/logql/engine.go

```diff
@@ -39,6 +41,11 @@ func (Streams) String() string {
 	return ""
 }
 
+type Result struct {
```

Comment:
Should this have a godoc comment?

Reply:
yes weird the linter has not cached this.